### ISSUE TITLE
BUG - Sev-2 - Android - Increase in auth failures

### ISSUE THREAD

**Issue by narin** [department-of-veterans-affairs/va-mobile-app#6446]
## What happened?

On Aug 2nd, [SIS reported an increase in auth failures](https://dsva.slack.com/archives/C03DY3FN7HQ/p1691012706671579), particularly for Android.  This co-insides with our release v2.8.0 which includes Custom Tabs for Android on August 1st.

After entering their MFA code, the user is taken to a blank white screen in the browser.  Upon refreshing, the error that pops up is `Code in state is not valid`.

We need to investigate whether the bug is limited to Custom Tabs, or if it might be an issue on SIS's end. 

## Specs:

- OS: Android

## Steps to Reproduce

1. Using an Android build of of 775 or later, attempt to log in using ID.me
2. After the "fake" 2FA confirmation, hit continue
3. Occasionally it will result in a white screen
4. Attempt to refresh the page to see the `Code in state is not valid` error.

## Desired behavior

User should be redirected to the app after hitting Continue upon entering their 2FA code

## Acceptance Criteria

- [ ] User no longer encounters white screen after completing auth, either via a bug fix on our end, a bug fix on SIS's end, or a rollback to remove Custom Tabs

## Bug Severity - BE SURE TO ADD THE SEVERITY LABEL
 
See [Bug Tracking](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/QA#issue-severity) for details on severity levels

- Impact: High
- Frequency: Low 

- 2 - High

## Screen shot(s) and additional information

<details>
  <summary>Full JSON response for services related to issue (expand/collapse)</summary>
  

</details>

**Comment by narin** [comment]
Some updates:
- We've determined that this bug would affect users who are on version 99 of Chrome on Android or below.  Version 100 was released in March of 2022.
- On Mon, Aug 7, we tried to convince the SIS team to add a "Continue" link so that users experiencing this bug would be unblocked
- They were resistant to this idea and instead tried to test a Backend 302 redirect, but it turns out this was not compatible with Login.gov due to a CORs issue
- On Thu, Aug 10, I attempted to add staging-api.va.gov as an intent-filter as a way to get around the `Navigation is blocked` error. This solution did not work as it still requires a user click to launch the app, otherwise it redirects to the URL within the browser
